common/browser_context.go

```diff
@@ -84,8 +84,11 @@ func (b *BrowserContext) AddInitScript(script goja.Value, arg goja.Value) error
 	_, isCallable := goja.AssertFunction(script)
 	hasArgs := gojaValueExists(arg)
 
-	var err error
-	var source string
+	var (
+		err    error
+		source string
+	)
```

Comment:
@ka3de You could do this refactoring in [this commit](https://github.com/grafana/xk6-browser/pull/719/commits/a91153d75c12579d785881bbf359464264e9c799) as it's a minor change. Refactoring/change commits are mostly needed when things can be confusing (for example, removing the switch cases and readding them in functions in a subsequent commit—as I explained in another comment).